src/automated-security-updates.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: all
+  name: Configure for automated security updates
```

Comment:
Is there a reason this is separate from the already included https://github.com/cisagov/ubuntu-server-packer/blob/126d189998f77db05e34052ba44e346127b6a1ff/src/base.yml#L7-L9?